tests/scripts/analyze_outcomes.py

```diff
@@ -214,6 +214,324 @@ def do_analyze_driver_vs_reference(results: Results, outcome_file, args):
             }
         }
     },
+    'analyze_driver_vs_reference_cipher_aead': {
+        'test_function': do_analyze_driver_vs_reference,
+        'args': {
+            'component_ref': 'test_psa_crypto_config_reference_cipher_aead',
+            'component_driver': 'test_psa_crypto_config_accel_cipher_aead',
+            # Ignore suites that are being accelerated
+            'ignored_suites': [
+                'aes.cbc',
+                'aes.cfb',
+                'aes.ecb',
+                'aes.ofb',
+                'aes.rest',
+                'aes.xts',
```

Comment:
Looks like we should add the ability to list `'aes'` and have all `'aes.xxx'` ignored. Actually, don't we already have this ability? Can't we replace these lines with just `'aes'`, and so on below?

Reply:
Ah, good point, I completely forgot we already had this ability even though I re-read that part of the code recently.